btree/btree.c

```diff
@@ -1908,10 +1947,28 @@ static int64_t tree_get(struct node_op *op, struct segaddr *addr, int nxt)
 static int64_t tree_create(struct node_op *op, struct m0_btree_type *tt,
 			   int rootshift, struct m0_be_tx *tx, int nxt)
 {
-	return segops->so_tree_create(op, tt, rootshift, tx, nxt);
+	struct td *tree;
+
+	/**
+	 * Creates root node and then assigns a tree descriptor for this root
+	 * node.
+	 */
+
+	tree_get(op, NULL, nxt);
+
```

Comment:
error handling, as tree_get can return error code

Reply:
This routine has not been touched yet as it is under consideration.
I will update once changes are to be made here.